.github/actions/javascript/postTestBuildComment/postTestBuildComment.js

```diff
@@ -0,0 +1,74 @@
+const core = require('@actions/core');
+const {context} = require('@actions/github');
+const GithubUtils = require('../../../libs/GithubUtils');
+
+const PR_NUMBER = core.getInput('PR_NUMBER', {required: true});
+
+const androidSuccess = core.getInput('ANDROID', {required: true}) === 'success';
+const desktopSuccess = core.getInput('DESKTOP', {required: true}) === 'success';
+const iOSSuccess = core.getInput('IOS', {required: true}) === 'success';
+const webSuccess = core.getInput('WEB', {required: true}) === 'success';
+
+const androidLink = androidSuccess
+    ? core.getInput('ANDROID_LINK')
+    : '❌ FAILED ❌';
+const desktopLink = desktopSuccess
+    ? core.getInput('DESKTOP_LINK')
+    : '❌ FAILED ❌';
```

Comment:
Let's use the same style every where for consistency. It's still easy to read on one line, IMO, but feel free to change `iOSLink` and `webLink` on different lines if you prefer instead.

```suggestion
const androidLink = androidSuccess ? core.getInput('ANDROID_LINK') : '❌ FAILED ❌';
const desktopLink = desktopSuccess ? core.getInput('DESKTOP_LINK') : '❌ FAILED ❌';
```